vng_api_common/notifications/constants.py

```diff
@@ -1,3 +0,0 @@
-# Exernally defined scopes.
-SCOPE_NOTIFICATIES_CONSUMEREN_LABEL = "notificaties.consumeren"
```

Comment:
I understand that these labels are duplicated now in `notifications-api-common`, but it looks like cleaning them is out of scope of this PR
Maybe a separate PR or at least a warning in CHANGELOG would be useful, because it could really be a breaking change

Reply:
I checked Open Zaak and Open Notificaties, the former already imports them from `notifications-api-common` and the latter doesn't use them. I'll add a warning in the changelog nonetheless